lib/ReactViewModels/ViewState.ts

```diff
@@ -24,6 +22,7 @@ import {
   TourPoint
 } from "./defaultTourPoints";
 import DisclaimerHandler from "./DisclaimerHandler";
+import NotificationState from "./NotificationState";
```

Comment:
There is still a `Notification` interface in this file

Reply:
(at line 43 - it won't let me comment on it)

Reply:
https://github.com/TerriaJS/terriajs/blob/5b7d0155473d3537b7a2402a6fe40566080113cc/lib/ReactViewModels/ViewState.ts#L42-L53